packages/slice-machine/tests/project/customtypes/about/index.json

```diff
@@ -23,6 +23,38 @@
         },
         "type": "Boolean"
       },
+      "myGroup": {
```

Comment:
We never push on the test project, but since it's going to be remove, we don't care.

Reply:
ok